### PR TITLE
kodi: update to 19.0

### PR DESCRIPTION
**Pull request by eatradish**
Topic Description
-----------------

kodi: update to 19.0

Package(s) Affected
-------------------

kodi: 19.0
waylandpp: 0.2.8
spdlog: 0.8.2
libudfread: 1.1.1
kodi-pvr-iptvsimple: 7.4.3

Security Update?
----------------

Build Order
-----------

{waylandpp,spdlog,libudfread:} -> kodi -> kodi-pvr-iptvsimple

Architectural Progress
----------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

Secondary Architectural Progress
--------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

----

After the pull request is merged, all package(s) affected must be rebuilt against the `stable` Git tree and environment (only `stable` repository should be enabled in `sources.list`). This section marks the progress above.

Please, make sure the list of architectures below matches the ones above.

Post-Merge Architectural Progress
---------------------------------

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

Post-Merge Secondary Architectural Progress
-------------------------------------------

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [ ] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

**Comment by eatradish** [comment]
LGTM on AMD64

Screenshot_20210308_001927

**Comment by RedL0tus** [comment]
@eatradish 好活当赏

**Comment by eatradish** [comment]
Screenshot_20210308_232331

iptvsimple LGTM

**Comment by eatradish** [comment]
LGTM on ARM64

File: extra-libs/libudfread/autobuild/defines [hunk 1]
```diff
@@ -0,0 +1,3 @@
+PKGNAME=libudfread
+PKGDES="A UDF Reader library"
+PKGSEC="libs"
```
Comment (MingcongBai) [inline extra-libs/libudfread/autobuild/defines:4]: No dependency whatsoever?

File: extra-libs/waylandpp/autobuild/defines [hunk 2]
```diff
@@ -0,0 +1,5 @@
+PKGNAME="waylandpp"
+PKGDES="Wayland C++ bindings"
+PKGDEP="wayland pugixml"
+BUILDDEP="cmake"
```
Comment (MingcongBai) [inline extra-libs/waylandpp/autobuild/defines]: This is unnecessary.

File: extra-libs/waylandpp/spec [hunk 3]
```diff
@@ -0,0 +1,4 @@
+VER=0.2.8
+SRCS="tbl::https://github.com/NilsBrause/waylandpp/archive/$VER.tar.gz"
+CHKSUMS="sha256::e7f486165d3568c3558b5c7099133aea4a285b82820eeafad329fc10271c654d"
+
```
Comment (MingcongBai) [inline extra-libs/waylandpp/spec]: Extra line here.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Pull request merged**